cookie-banner-rules-list.json

```diff
@@ -2118,18 +2063,21 @@
       "domains": ["meteo.be"]
     },
     {
-      "click": { "optIn": "a.wscrOk", "presence": "div.wscrBannerContent" },
+      "click": {
+        "optIn": "a.wscrOk",
+        "presence": "div.wscrBannerContent"
+      },
       "cookies": {
         "optIn": [
           {
             "name": "wscrCookieConsentShared",
-            "value": "1=true&2=true&3=true&4=true&5=true&visitor=03bfbe89-44d7-4da3-ad3f-a1a8a8285257&version=20221005-001"
+            "value":"1=true&2=true&3=true&4=true&5=true&visitor=44dd8519-4cd6-46ab-bf7d-7988c112da26&version=20231107-001"
           }
         ],
         "optOut": [
           {
             "name": "wscrCookieConsentShared",
-            "value": "1=true&2=false&3=false&4=false&5=false&visitor=5698152c-e2fd-4076-b411-f8baed432e40&version=20221005-001"
+            "value": "1=true&2=false&3=false&4=false&5=false&visitor=44dd8519-4cd6-46ab-bf7d-7988c112da26&version=20231107-001"
```

Comment:
We shouldn't inject cookies with uuids since they are likely to be unstable. Let's rely only on clicking instead.